.dockerignore

```diff
@@ -1,6 +1 @@
-*
```

Comment:
@dgageot prefers the "Ignore everything first and then add things that are needed one by one"